pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -1153,3 +1157,11 @@ message MemoryUsage {
     // The amount of working set memory in bytes.
     UInt64Value working_set_bytes = 2;
 }
+
+message ReopenContainerLogRequest {
+    // ID of the container for which to retrieve status.
```

Comment:
typo: s/to retrieve status/to reopen container log file/.

Reply:
Done.